Frontend/CO.CDP.OrganisationApp/Pages/Supplier/LegalFormFormationDate.cshtml.cs

```diff
@@ -18,18 +19,18 @@ public class LegalFormFormationDateModel(
     public Guid Id { get; set; }
 
     [BindProperty]
-    [Required(ErrorMessage = "Date of registration must include a day")]
-    [RegularExpression(RegExPatterns.Day, ErrorMessage = "Day must be a valid number")]
+    [Required(ErrorMessage = nameof(@StaticTextResource.Supplier_LegalFormFormationDate_DayErrorMessage))]
+    [RegularExpression(RegExPatterns.Day, ErrorMessage = nameof(@StaticTextResource.Supplier_LegalFormFormationDate_DayInvalidErrorMessage))]
     public string? Day { get; set; }
 
     [BindProperty]
-    [Required(ErrorMessage = "Date of registration must include a month")]
-    [RegularExpression(RegExPatterns.Month, ErrorMessage = "Month must be a valid number")]
+    [Required(ErrorMessage = nameof(@StaticTextResource.Supplier_LegalFormFormationDate_MonthErrorMessage))]
+    [RegularExpression(RegExPatterns.Month, ErrorMessage = nameof(@StaticTextResource.Supplier_LegalFormFormationDate_MonthInvalidErrorMessage))]
     public string? Month { get; set; }
 
     [BindProperty]
-    [Required(ErrorMessage = "Date of registration must include a year")]
-    [RegularExpression(RegExPatterns.Year, ErrorMessage = "Year must be a valid number")]
+    [Required(ErrorMessage = nameof(@StaticTextResource.Supplier_LegalFormFormationDate_YearErrorMessage))]
+    [RegularExpression(RegExPatterns.Year, ErrorMessage = nameof(@StaticTextResource.Supplier_LegalFormFormationDate_YearInvalidErrorMessage))]
```

Comment:
Please switch to `ErrorMessageResourceName` and `ErrorMessageResourceType` as found elsewhere in validation attributes. What you've done _was_ correct, but we had to update the pattern elsewhere as it didn't work in some circumstances with custom validators. Moving to this pattern allows us to keep it the same everywhere. Translating ErrorMessage directly like this is discouraged by the .Net docs as well

Reply:
Done